packages/core/lifecycle/core-lifecycle-server-internal/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@kbn/core-lifecycle-server-internal",
+  "private": true,
+  "version": "1.0.0",
+  "main": "./target_node/index.js",
+  "license": "SSPL-1.0 OR Elastic License 2.0"
```

Comment:
NIT: missing `"author": "Kibana Core",` (in other packages too)

Reply:
really? I entered our team at the prompt for adding one.
cc @elastic/kibana-operations is there a bug in the package generation script? I created these using
`node scripts/generate package --dir ./packages/core/plugins/core-plugins-server-internal  @kbn/core-plugins-server-internal` and when prompted for a team name, entered "@elastic/kibana-core"

Reply:
We've been adding this line manually in our packages. Now sure exactly why though...

Reply:
oh, right. Thanks for the reminder!